middlewares/checkDb.js

```diff
@@ -0,0 +1,32 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import models from '../models';
+
+dotenv.config();
+
+const { users } = models;
+
+export const checkAdmin = async (req, res, next) => {
+  const authHeader = req.headers.authorization;
+  const token = authHeader.split(' ')[1];
+  const decode = jwt.decode(token, process.env.SECRET);
+  const { email } = decode;
+  const foundUser = await users.findOne({
+    where: {
+      email
+    },
+    attributes: ['accessLevel']
+  });
+
+  switch (true) {
+    case foundUser === null:
+      return res.status(404).json({
+        error: 'failed to access resource'
+      });
+    case foundUser < 1:
```

Comment:
levels are `unsigned` ints which means theres no room for `signed` ints like `negatives` change your comparison so that it doesnot go beyond zero. 
maybe if you have a good reason to let it go beyond a zero. Any check it with negatives and see the outputs. Thanks